### PR TITLE
DOC Add Provisioning with dashboards from grafana.com

### PR DESCRIPTION
**Pull request by mfaure**
#### Pull Request (PR) description

Add explanations on how to provision dashboards coming from Grafana.com

### CODE REVIEW

**Comment by mfaure** [comment]
@bastelfreak @Dan33l Following our discussions on IRC, I found out a way to provision dashboards coming from Grafana.com.

Would you kindly review ?

**Comment by bastelfreak** [comment]
Thanks for the PR @mfaure. Please have a look at the travis issues.

File: README.md [hunk 1]
```diff
@@ -883,6 +883,43 @@ the filesystem of the puppetserver/master. Thus you may specify a
 local directory with grafana dashboards you wish to provision into
 grafana.
 
+##### Provisioning with dashboards from grafana.com
+
```
[the remark this answers is not in the document]
Comment (mfaure) [reply README.md]: In fact "outside the code" is correct, but it is more how to *prepare* the JSON file that will be included in the code (as a file or template). I add a few words about this.

**Comment by mfaure** [comment]
@bastelfreak @Dan33l Thanks for your feedback. Travis is OK. I've had a few more words to precise the steps are outside the code. Feel free to amend :)

**Pull request merged**